client/watcher/watcher.go

```diff
@@ -89,6 +98,16 @@ func (w *Watcher) Watch() {
 						printer.LogErrf("fsnotify Add error: %v", err)
 					}
 				}
+			case <-timer.C:
+				err := w.UpdateBytes()
+				if err != nil {
+					printer.LogErrf("UpdateBytes error: %v", err)
+				}
+				// Rewatch the path
+				err = w.watcher.Add(w.path)
```

Comment:
Do we need to rewatch the path in this case?
It does look to cause no trouble, so maybe ok to leave. But it does seem that we could do without.

I do wonder about the `Add()` on line 87. I don't see anything in the doc about needing to rewatch. But I believe this was something we discovered was necessary to make it work. Outside the scope of this PR, but could maybe use a comment.

Reply:
This should be safe to do. I was worried about it myself, so did some digging and discovered it was added in [this PR](https://github.com/vsco/dcdr/pull/60/files#diff-e36200014dddcb7f016acdc8db7bc9a0R86-R89). [`Add` ultimately invokes inotify_add_watch syscall](https://github.com/fsnotify/fsnotify/blob/v1.4.7/inotify.go#L109), which is idempotent: http://man7.org/linux/man-pages/man2/inotify_add_watch.2.html

I added it here, to ensure that a filesystem change (move/removing directories), doesn't cause the watch to get removed.